.travis.yml

```diff
@@ -35,18 +35,18 @@ install:
   - conda info -a
 
   # create environment and install dependencies
-  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy scipy nose pip matplotlib coverage
+  - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION numpy>=1.15.0 scipy nose pip matplotlib coverage
   - source activate test-environment
-  - conda install -c conda-forge healpy aipy scikit-learn
+  - conda install -c conda-forge healpy aipy
```

Comment:
hera_qm and hera_cal, while not strictly required, have some advanced functionality that relies on `scikit-learn`, so it'd be nice to keep this if possible. was something breaking with the installation?

Reply:
Thanks Nick. The reason from removing `scikit-learn` from this line is that it was pulling in an old version of `numpy`, which is now incompatible with `pyuvdata` (thus causing the tests to fail). It *is* installed via `pip` later on in the Travis script however (pip allows it to install with a more recent version of `numpy`).